fix: claude-review GitHub Action fails — claude executable not found in CI runner PATH

## Context

The `claude-review` GitHub Action check fails on every PR with:

```
Action failed with error: Failed to add marketplace 'https://github.com/anthropics/claude-code.git': Executable not found in $PATH: "claude"
```

This is a CI infrastructure issue — the `claude` CLI is not installed or not on PATH in the GitHub Actions runner environment.

## Evidence

From PR #966 CI run (run ID `23309180877`):

```
##[error]Action failed with error: Failed to add marketplace
'https://github.com/anthropics/claude-code.git':
Executable not found in \$PATH: "claude"
```

## Impact

- Every PR shows a red `claude-review` check, even when the code is correct
- The review loop (`review_driver.py`) treats this as a CI failure and stops (`STOPPED_CI_FAILURE`)
- Auto-merge cannot proceed because the check fails

## Likely Fix

One of:
1. Add a `claude` installation step to the `claude-code-review.yml` workflow (e.g., `npm install -g @anthropic-ai/claude-code`)
2. Set `ANTHROPIC_API_KEY` secret in the repo so the action can authenticate
3. If the action is not intended to run (no API key configured), mark the check as optional / not required in branch protection, or add a `continue-on-error: true` to the job

## References

- Workflow file: `.github/workflows/claude-code-review.yml`
- Failed run: https://github.com/Questuart/Bid-Euchre/actions/runs/23309180877
- Affected PRs: #966 (and likely all recent PRs)

--- a/.github/workflows/claude-code-review.yml
+++ b/.github/workflows/claude-code-review.yml
@@ -36,8 +36,8 @@ jobs:
         uses: anthropics/claude-code-action@v1
         with:
           claude_code_oauth_token: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN }}
-          plugin_marketplaces: 'https://github.com/anthropics/claude-code.git'
-          plugins: 'code-review@claude-code-plugins'
-          prompt: '/code-review:code-review ${{ github.repository }}/pull/${{ github.event.pull_request.number }}'
-          # See https://github.com/anthropics/claude-code-action/blob/main/docs/usage.md
-          # or https://code.claude.com/docs/en/cli-reference for available options
+          prompt: |
+            Review this pull request for code quality, correctness, and security.
+            Analyze the diff, then post your findings as review comments.
+            Follow the project's CLAUDE.md guidelines.
+          claude_args: "--max-turns 5"

--- a/tests/unit/test_claude_review_workflow.py
+++ b/tests/unit/test_claude_review_workflow.py
@@ -1,0 +1,51 @@
+"""Regression tests for .github/workflows/claude-code-review.yml structure."""
+
+from __future__ import annotations
+
+from pathlib import Path
+
+import yaml
+
+WORKFLOW = (
+    Path(__file__).resolve().parents[2] / ".github/workflows/claude-code-review.yml"
+)
+
+
+class TestClaudeReviewWorkflow:
+    """Validate claude-code-review.yml has the expected structure."""
+
+    def setup_method(self):
+        self.cfg = yaml.safe_load(WORKFLOW.read_text())
+
+    def test_uses_v1_action(self):
+        step = self._review_step()
+        assert step["uses"] == "anthropics/claude-code-action@v1"
+
+    def test_has_oauth_token(self):
+        step = self._review_step()
+        with_block = step["with"]
+        assert "claude_code_oauth_token" in with_block
+
+    def test_no_plugin_marketplaces(self):
+        """plugin_marketplaces causes 'claude not found' errors on runners."""
+        step = self._review_step()
+        with_block = step["with"]
+        assert "plugin_marketplaces" not in with_block
+        assert "plugins" not in with_block
+
+    def test_has_prompt(self):
+        step = self._review_step()
+        assert "prompt" in step["with"]
+
+    def test_has_max_turns(self):
+        step = self._review_step()
+        assert "claude_args" in step["with"]
+        assert "--max-turns" in step["with"]["claude_args"]
+
+    def _review_step(self):
+        """Return the 'Run Claude Code Review' step."""
+        steps = self.cfg["jobs"]["claude-review"]["steps"]
+        for s in steps:
+            if s.get("id") == "claude-review":
+                return s
+        raise AssertionError("claude-review step not found")

